template/composer.json

```diff
@@ -57,11 +57,14 @@
   },
   "scripts": {
     "install-phantomjs": "PhantomInstaller\\Installer::installPhantomJS",
+    "install-blt-alias": "./vendor/acquia/blt/scripts/blt/install-alias.sh -y",
```

Comment:
call this with `yes | ./vendor/acquia/blt/scripts/blt/install-alias.sh` to accept `y` input.